lib/crystalball.rb

```diff
@@ -16,6 +16,7 @@
 require 'crystalball/map_generator/coverage_strategy'
 require 'crystalball/map_generator/allocated_objects_strategy'
 require 'crystalball/map_generator/described_class_strategy'
+require 'crystalball/map_generator/parser_strategy'
```

Comment:
Bad idea to require it by default, it would make `parser` a required dependency.